api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -144,18 +149,15 @@ enum BlobStatus {
   // ENCODED means that the blob has been Reed-Solomon encoded into chunks and is ready to be dispersed to DA Nodes.
   ENCODED = 2;
 
-  // CERTIFIED means the blob has been dispersed and attested by the DA nodes.
-  CERTIFIED = 3;
+  // GATHERING_SIGNATURES means that the blob has been dispersed and is waiting for the attestations from DA network.
```

Comment:
"dispersed" -> "dispatched to validators"?
"dispersed" is a bit overloaded, people think sending blob to Disperser is also "dispersed".